.gitignore

```diff
@@ -1,3 +1,4 @@
+.vscode/settings.json
```

Comment:
This is something that is specific to your personal development setup. It would be better to add it to your global gitignore file rather than each project you work on.


```
git config core.excludesfile ~/.gitignore
```

and add things like this then to `~/.gitignore`